packages/tables/src/styled/StyledTable.ts

```diff
@@ -0,0 +1,47 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import styled, { ThemeProps, DefaultTheme } from 'styled-components';
+import { retrieveComponentStyles, DEFAULT_THEME } from '@zendeskgarden/react-theming';
+
+const COMPONENT_ID = 'tables.table';
+
+export type SIZE = 'small' | 'medium' | 'large';
+
+export interface IStyledTableProps {
+  size?: SIZE;
+}
+
+export const getLineHeight = (props: ThemeProps<DefaultTheme>) => {
+  return `${props.theme.space.base * 5}px`;
+};
+
+/**
+ * 1. <table> reset
+ * 2. <th> reset
```

Comment:
I don't see the cross-reference for `[2]` below.

Reply:
Removed.